fluentd/run.sh

```diff
@@ -105,5 +105,8 @@ else
     echo > $CFG_DIR/dynamic/es-ops-copy-config.conf
 fi
 
-
-exec fluentd $fluentdargs
+if [[ $DEBUG ]] ; then
+    exec fluentd $fluentdargs > /var/log/fluentd.log 2>&1
```

Comment:
Why are we just writing our logs out to /var/log/fluentd.log if we're debugging? Is it possible to move this to the fluentdargs instead since we are adding `-vv` there.